lms/static/js/views/fields.js

```diff
@@ -201,13 +205,15 @@
                 this.$el.addClass('mode-edit');
             },
 
-            startEditing: function () {
+            startEditing: function (event) {
+                event.preventDefault();
```

Comment:
This will not be needed after: https://github.com/edx/edx-platform/pull/7898/files